mozzi_config.h

```diff
@@ -97,7 +97,7 @@ compile time.
       MONO is 1 and the value of STEREO is 2, so future extensions are also expected
       to set this to the number of available channels. */
 #define AUDIO_CHANNELS MONO
-//#define AUDIO_CHANNELS STEREO
+ //#define AUDIO_CHANNELS STEREO
```

Comment:
Nitpick, but please clean up that space, so it does not end up as a changed line.